src/interface/src/app/signup/signup.component.html

```diff
@@ -1,104 +1,126 @@
-        </mat-form-field>
+          <mat-label class="standard-label"
+            >Last Name
+            <span class="required-blurb">required</span>
+          </mat-label>
+          <mat-form-field appearance="outline">
+            <input type="text" required formControlName="lastName" matInput />
+          </mat-form-field>
+          <app-field-alert
+            *ngIf="
+              this.form.controls['lastName'].errors !== null &&
+              this.form.controls['lastName'].dirty
+            "
+            message="This field must contain a value."
+            title="Last Name Error"></app-field-alert>
 
-        <mat-form-field appearance="fill">
-          <mat-label>Email</mat-label>
-          <input type="text" required formControlName="email" matInput />
-        </mat-form-field>
+          <mat-label class="standard-label"
+            >Email
+            <span class="required-blurb">required</span>
+          </mat-label>
+          <mat-form-field appearance="outline">
+            <input
+              type="text"
+              required
+              formControlName="email"
+              matInput
+              (blur)="getEmailError()" />
+          </mat-form-field>
+          <app-field-alert
+            *ngIf="
+              this.form.controls['email'].touched &&
+              this.form.controls['email'].errors !== null
+            "
+            [message]="getEmailError()"
+            title="email error"></app-field-alert>
 
-        <mat-form-field appearance="fill">
-          <mat-label>Create password</mat-label>
-          <input
-            type="password"
-            required
-            formControlName="password1"
-            matInput />
-          <mat-error *ngIf="form.get('password1')?.hasError('minlength')">
-            Password must contain at least 8 characters.
-          </mat-error>
-        </mat-form-field>
+          <mat-label class="standard-label"
+            >Create Password
+            <span class="required-blurb">required</span>
+          </mat-label>
+          <mat-form-field appearance="outline">
+            <input
+              type="password"
+              required
+              formControlName="password1"
+              matInput
+              autocomplete="password"
+              [errorStateMatcher]="passwordStateMatcher"
+              (focus)="showHint = true"
+              (blur)="showHint = false" />
+          </mat-form-field>
+          <app-field-alert
+            [messageType]="FormMessageType.ALERT"
+            *ngIf="showHint && !form.get('password1')?.value"
+            title="Password requirements"
+            message="Your password needs to be at least 8 characters long.">
+          </app-field-alert>
+          <app-field-alert
+            [messageType]="FormMessageType.ERROR"
+            *ngIf="
+              (!showHint && form.get('password1')?.errors?.['minlength']) ||
```

Comment:
can you do `form.get('password').invalid` instead of checking for specific errors?

Reply:
It actually makes no sense for this to check the newPasswordMustBeNew condition anyway, since we don't have a Current Password field on this screen.